contracts/01-tamagotchi/src/lib.rs

```diff
@@ -1,19 +1,46 @@
 #![no_std]
 
 #[allow(unused_imports)]
-use gstd::prelude::*;
+use gstd::{prelude::*,msg,exec};
```

Comment:
Prefer using `rustfmt` before committing.
```suggestion
use gstd::{exec, msg, prelude::*};
```

Reply:
Thank you.